Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan.yml

```diff
@@ -2456,7 +2449,7 @@ inputs:
   description: Set to true to assign only the users that are currently on shift.
   playbookInputQuery:
 outputs: []
-fromversion: 5.0.0
+fromversion: 6.0.0
```

Comment:
Check why 5 became 6

Reply:
I believe I got a validation error somewhere (I don't recall where, I tried running `validate` and I can't seem to reproduce the error), but basically one of the sub-playbooks used on this playbook has `fromversion: 6.0.0`, which means it required version 6.0.0 to work?

`Isolate Endpoint - Generic V2` for example has `fromversion: 5.5.0`.

Reply:
This happens when the hosting playbook calls for a sub-playbook with higher versioning.
It can be skipped using the `skip if unavailable` option.
Anyway, if that is what the validation did - and the new playbook is from 6.0 -> let's keep this change.